index.bs

```diff
@@ -682,30 +682,25 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |clientDataHash| be the [=hash of the serialized client data=] represented by |clientDataJSON|.
 
-1. Let |currentlyAvailableAuthenticators| be a new [=ordered set=] consisting of all [=authenticators=]
-    currently available on this platform.
+1. Let |lifetimeTimer| be a timer for |adjustedTimeout| milliseconds.
```

Comment:
Do we need to add a new name? We can just change adjustedTimeout to lifetimeTimer. 

Note: this is more of a nitpick. Feel free to merge the PR without changing the adjustedTimeout naming.